packages/resource-deployment/scripts/upload-files.sh

```diff
@@ -68,4 +57,4 @@ uploadFolderContents $jobManagerContainerName "$dropFolder/job-manager/dist" "$s
 uploadFolderContents $runnerContainerName "$dropFolder/runner/dist" "$storageAccountName" "$includePattern"
 uploadFolderContents $scanRequestSenderContainerName "$dropFolder/scan-request-sender/dist" "$storageAccountName" "$includePattern"
 uploadFolderContents $poolStartupContainerName "$dropFolder/resource-deployment/dist/scripts/pool-startup" "$storageAccountName" "$includePattern"
-uploadFile $runtimeConfigurationContainerName "$dropFolder/resource-deployment/dist/runtime-config/runtime-config.$environment.json" "$storageAccountName" "$runtimeConfigurationBlobName"
+. "${0%/*}/upload-config-files.sh"
```

Comment:
Will this script need to be passed some parameters or is there another way that it gets these parameters (env variables etc)?

Reply:
when you call a script inside another using the same way . "filepath"
the second script gets executed inside the first script process, all function and variables in the first script are accessible in the second, so it will use the same input parameters for the first script.